release_collection.py

```diff
@@ -409,7 +409,7 @@ def role_to_collection(
         "--role",
         rolename,
         "--src-path",
-        args.src_path,
+        os.path.join(args.src_path, rolename),
```

Comment:
`src_path` can be either the parent directory of the role, or the role directory.  Forcing it to be only the role directory will break some of our use cases.

Reply:
@richm Okay - so what be an acceptable solution then? The only other thing that currently comes to my mind is, to checkout remote roles to individual tmp folders. As you can see in the provided error log in the issue #357 you code currently picks up the wrong folder on the second cloned repository.

Reply:
@hhenkel What repo(s) are `release_collection.py` checking out from github?  What does your `collection_release.yml` look like?

Reply:
It looks like this:
```
usergroups:
  org: Github-My_corp-CHE
  repo: cioapir-my_corp.ansible-role-usergroups
  ref: 0.0.0
  sourcenum: 1
users:
  org: Github-My_corp-CHE
  repo: cioapir-my_corp.ansible-role-users
  ref: 0.0.0
  sourcenum: 2
sudo:
  org: Github-My_corp-CHE
  repo: cioapir-my_corp.ansible-role-sudo
  ref: 0.0.0
  sourcenum: 3
```